### DISCUSSION TITLE
Function request: save the image in interval on every first of the month at midnight - on SD or FTP...

### DISCUSSION THREAD

**Discussion by TBobsin** [jomjol/AI-on-the-edge-device#2400] [Ideas]
Hi,
i want to save the Picture that is taken at some interval 
Meanly the first day of an month at midnight, or daily every midnight. 

It could be saved on the SD card (once a month or day is not the problem) or send it to an file storage over ftp, NFS, WebDav ...
Alternative share the picture, so that it could be downloaded. 

Monthly pictures could be deleted after 18 month, 
Delete daily pictures? I this it depends on the size of the SD, keeping a week seems good. 

I think this request has the same reason
https://github.com/jomjol/AI-on-the-edge-device/discussions/1330

**Comment by adahmen** [comment]
I'm also looking for this feature. Sometimes the detected value is not correct or the digitization is not working - so the image would be very helpful.

**Comment by caco3** [comment]
I added a feature request to it: https://github.com/jomjol/AI-on-the-edge-device/issues/3326

Note that this does not mean that somebody is working on it!
